CommunityToolkit.Net.Graph/CommunityToolkit.Net.Graph.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Title>Community Toolkit .NET Standard Graph Services</Title>
+    <PackageId>CommunityToolkit.Net.Graph</PackageId>
+    <Description>
+      This package includes .NET Standard code helpers such as:
+      - GraphExtensions: Helpers for common tasks related to the Microsoft Graph used by the Microsoft.Toolkit.Graph.Controls.
+      - ProviderExtensions: Extension on IProvider for accessing a pre-configured GraphServiceClient instance.
+    </Description>
+    <PackageTags>Community Toolkit Graph Provider Extensions</PackageTags>
+    <SignAssembly>false</SignAssembly>
+    <GenerateLibraryLayout>true</GenerateLibraryLayout>
+    <LangVersion>8.0</LangVersion>
+    <Configurations>Debug;Release;CI</Configurations>
+    <Platforms>AnyCPU;ARM;ARM64;x64;x86</Platforms>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Graph.Beta" Version="0.39.0-preview" />
```

Comment:
Just a thought - should we have two packages (one pulling in the beta sdk, and one the v1 sdk)? The source can be all the same, just this csproj will be different.

Reply:
That sounds pretty slick. Do you think there is any potential for type mismatch if the SDKs deviate much?

Michael and I had talked about splitting on Beta vs v1, this could be a great way to do it. Either way, I'll save that for another PR - #52

Reply:
Awesome. Yeah, I think it's worth trying.